src/fineman/elimination_algorithm.py

```diff
@@ -0,0 +1,91 @@
+from math import ceil
+
+from fineman.betweenness_reduction import betweenness_reduction
+from fineman.elimination_by_hop_reduction import elimination_of_r_remote_edges_by_hop_reduction
+from fineman.helper_functions import reweight_graph, super_source_bfd, compute_reach, transpose_graph, b_hop_sssp, \
+    b_hop_stsp
+from fineman.independent_set_or_crust import find_is_or_crust
+
+
+def _eliminate_independent_set(graph, independent_set):
```

Comment:
is this name appropriate for a function that doesn't eliminate anything but only returns a price function that if applied eliminates the 1-hop independent set?